content/guides/zapier-neon.md

```diff
@@ -0,0 +1,250 @@
+  - A table named `form_submissions` to demonstrate adding data from an external source.
+
+    You can create these tables using the [Neon SQL Editor](/docs/get-started-with-neon/query-with-neon-sql-editor) or any Postgres client such as [`psql`](/docs/connect/query-with-psql-editor)
+
+    **Example SQL for `users` table:**
+
+        ```sql
+        CREATE TABLE users (
+            id SERIAL PRIMARY KEY,
+            name VARCHAR(255),
+            email VARCHAR(255) UNIQUE,
+            signed_up_at TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP
+        );
+        ```
+
+    **Example SQL for `form_submissions` table:**
+
+        ```sql
+        CREATE TABLE form_submissions (
+            id SERIAL PRIMARY KEY,
+            submitter_email VARCHAR(255),
+            feedback_text TEXT,
+            submitted_at TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP
+        );
+        ```
+
+<Admonition type="important">
+For Zapier's "New Row" trigger to reliably detect new entries in Neon, your table should have an auto-incrementing `PRIMARY KEY` (like `SERIAL` or `BIGSERIAL`) or a column that strictly orders new rows (like a `created_at` timestamp). Zapier uses this "Ordering Column" to check for new entries.
+</Admonition>
+
+## Connecting Neon Postgres to Zapier
+
+Before creating Zaps, you need to connect your Neon database to Zapier. Zapier uses a generic "PostgreSQL" app integration. Due to how [Neon uses Server Name Indication (SNI) for routing connections](/docs/connect/connection-errors#the-endpoint-id-is-not-specified) and how some clients handle SNI, a specific format is required for the password field in Zapier to ensure a successful connection.
+
+1.  **Log in to Zapier.**
+2.  Navigate to "**App Connections**" from the left sidebar.
+    ![App Connections in Zapier](/docs/guides/zapier-app-connections.png)
+3.  Click "**Add connection**" and search for "**PostgreSQL**".
+    ![Add connection page in Zapier](/docs/guides/zapier-add-connection.png)
+4.  A pop-up window will appear asking for connection details. You can find most of these in your Neon Console on the **Dashboard** page, by clicking on the **Connect** button for your database. Fill in the following fields:
+
+    - **Host:** Your Neon host (e.g., `ep-tight-boat-a6aplura-pooler.us-west-2.aws.neon.tech`)
+    - **Port:** `5432`
+    - **Database:** Your Neon database name (e.g., `neondb`)
+    - **Username:** Your Neon database user (e.g., `neon_user`)
+    - **Password:** **This is where the special format is needed.** See the important note below.
+
+    <Admonition type="important" title="Password Format for Neon Postgres in Zapier">
+    To connect Zapier to Neon successfully, you must include your Neon **Endpoint ID** within the password field. This is because Neon uses SNI to route connections, and some clients like Zapier's PostgreSQL connector donot pass SNI information in a way that Neon can use directly without this workaround.
```

Comment:
```suggestion
    To connect Zapier to Neon successfully, you must include your Neon **Endpoint ID** within the password field. This is because Neon uses SNI to route connections, and some clients like Zapier's PostgreSQL connector do not pass SNI information in a way that Neon can use directly without this workaround.
```